_includes/configuration/advtable-value-series.md

```diff
@@ -0,0 +1,82 @@
+
+**Type:** `Object`
+
+**Default Value:**
+ - `numbers` - a natural number series.
+ - `alpha` - a base26 letter series.
+
+```js
+{
+  numbers: {
+    update: true,
+    resizable: false,
+    generator: ...
+  },
+  alpha: {
+    update: true,
+    resizable: false,
+    generator: ...
+  },
+}
+```
+
+### Series configuration
+
+| Name | Value | Requirement | Description |
+| ---- | ----- | ----------- | ----------- |
+| update | `boolean` | Optional | default: `false` - Specifies whether the table cells that are part of the series should update after changes are made to the table. |
+| resizable | `boolean` | Optional | default: `true` - Specifies whether the table cells that are part of the series are resizable when using the resize bars. |
+| generator | `(info: GeneratorInfo, rowIndex: number, columnIndex: number) => GeneratorResult` | Required | [Usage of generator](#usageofgenerator). |
+
+#### Usage of generator
+
+The `generator` is a callback function that is used to specify how a table cell that is part of the series should be updated. The callback is passed information relating to the generator and table cell, the row index and column index of the table cell. For more details, see: [GeneratorInfo](#generatorinfo).
+The callback should return an object containing the value and optionally, the classes and attributes that should be applied on the table cell. For more details, see: [GeneratorResult](#generatorresult).
+
+##### GeneratorInfo
+
+| Name | Value | Description |
+| ---- | ----- | ----------- |
+| rowType | `'thead'`, `'tbody'` or '`tfoot'` | The section of the table cell. |
+| cellType | `'td'` or `'th'` | The type of the table cell. |
+| direction | `'row'` or `'column'` | The direction of the generator. |
+| prevSeriesValue | `string` or `undefined` | The previous value calculated from the generator. |
+
+##### GeneratorResult
+
+| Name | Value | Requirement | Description |
+| ---- | ----- | ----------- | ----------- |
+| classes | `string[]` | Optional | The classes that should be applied on the table cell. |
+| attributes | `Record<string, string, boolean, number or null>` | Optional | The attributes that should be applied or removed on the table cell. |
+| value | `string`, `number` or `undefined` | Optional | The value of the table cell. If the given value is `undefined`, it will attempt to use the previous value of the table cell. |
+
+### Example: Using `advtable_value_series`
+
+```js
+tinymce.init({
+  selector: 'textarea',  // change this value according to your html
+  plugins: 'table advtable',
+  menubar: 'table',
+  advtable_value_series: {
+    numbers: {
+      update: true,
+      resizable: false,
+      generator: function (info, rowIndex, columnIndex) {
+        return {
+          value: rowIndex + 1
+        };
+      }
+    },
+  }
+});
+```
+
+{% comment %} TODO: Do I need to make a note that this option is used in conjuction with the mceTableToggleSeries command? {% endcomment %}
```

Comment:
if they don't define advtable_value_series, and call mceTableToggleSeries, does it throw some kind of advtable_value_series is undefined?

Reading this on the one pager, its implied that we need that setting to make it work, on the actual website, that may not be so clear.  We could note that in advtable-cmds.md, requires setting advtable_value_series or throw advtable_value_series needs to be defined in the configuration

Reply:
oh the next advtable.md, says just that